streams/src/main/java/org/apache/kafka/streams/kstream/TimeWindowedDeserializer.java

```diff
@@ -61,6 +61,21 @@ public Long getWindowSize() {
     @SuppressWarnings("unchecked")
     @Override
     public void configure(final Map<String, ?> configs, final boolean isKey) {
+        //check if the config is set and the window size is already set from the constructor
+        final Long configWindowSize;
+        if (configs.get(StreamsConfig.WINDOW_SIZE_MS_CONFIG) instanceof String) {
```

Comment:
The console consumer made this check necessary - if there's a simpler way to do this lmk